addons/udes_common/models/ir_module.py

```diff
@@ -0,0 +1,16 @@
+from odoo import models, api
+
+
+class Module(models.Model):
+    _inherit = "ir.module.module"
+
+    @api.model
+    def get_module_installed(self, module_name):
```

Comment:
Nit: would is_module_installed be better here a ti just returns boolean, rather than the module

Reply:
Yeah does sound more suitable, will rename. :)

Reply:
Have now renamed this.